proxy/src/http/sql_over_http.rs

```diff
@@ -47,6 +47,7 @@ enum Payload {
 
 const MAX_RESPONSE_SIZE: usize = 10 * 1024 * 1024; // 10 MiB
 const MAX_REQUEST_SIZE: u64 = 10 * 1024 * 1024; // 10 MiB
+const HTTP_CONNECTION_TIMEOUT_SECS: u64 = 15;
```

Comment:
This could be just a duration instead:
```suggestion
const HTTP_CONNECTION_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(15);
```

Reply:
Done.